tests/hipscat/pixel_math/test_partition_stats.py

```diff
@@ -167,3 +168,42 @@ def test_destination_pixel_map_order1():
     result = hist.generate_destination_pixel_map(initial_histogram, alignment)
 
     npt.assert_array_equal(result, expected)
+
+
+def test_compute_pixel_map_order1():
+    """Create destination pixel map for small sky at order 1"""
+
+    initial_histogram = hist.empty_histogram(1)
+    filled_pixels = [51, 29, 51, 0]
+    initial_histogram[44:] = filled_pixels[:]
+
+    expected = {HealpixPixel(0, 11): (131, [44, 45, 46])}
+
+    result = hist.compute_pixel_map(initial_histogram, highest_order=1, threshold=150)
+
+    npt.assert_array_equal(result, expected)
+
+    expected = {
+        HealpixPixel(1, 44): (51, [44]),
+        HealpixPixel(1, 45): (29, [45]),
+        HealpixPixel(1, 46): (51, [46]),
+    }
+
+    result = hist.compute_pixel_map(initial_histogram, highest_order=1, threshold=100)
+
+    npt.assert_array_equal(result, expected)
+
+def test_compute_pixel_map_invalid_inputs():
+    """Create destination pixel map for small sky at order 1"""
+
+    initial_histogram = hist.empty_histogram(1)
+    filled_pixels = [51, 29, 51, 0]
+    initial_histogram[44:] = filled_pixels[:]
+
+    ## Order doesn't match histogram length
+    with pytest.raises(ValueError):
```

Comment:
you could check to make sure that each of these is raising the right `ValueError` with something like

```python
with pytest.raises(ValueError) as length_error:
    hist.compute_pixel_map(initial_histogram, highest_order=2, threshold=150)

assert str(length_error.value) == "histogram is not the right size"
```
and ditto for the next one